src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -496,27 +497,40 @@ public function loadByQuery(
     /**
      * {@inheritdoc}
      */
-    public function loadTreeByUuid(
+    public function loadDescendantsInclusive(
```

Comment:
We now remove both loadTreeByUUid and loadTreeByPath and replace them with this method which returns the given document and its *ancestors* (fix that) until the HomeDocument.